assets/scss/_styles_project.scss

```diff
@@ -103,10 +103,6 @@
   }
 }
 
-.foldable-nav {
```

Comment:
I think you should also remove the class from whatever HTML element has it attached?

Reply:
Sure.
I checked the repo.
It seems there is not any HTML element in this repo.